rest/src/config-create.h

```diff
@@ -27,7 +27,7 @@ namespace fty::disco {
 class ConfigCreate : public rest::Runner
 {
 public:
-    INIT_REST("discovery/config");
+    INIT_REST("discovery/config/create");
```

Comment:
config rest should be in admin/discovery/auto-discovery-config so not sure we need to have the create here

Reply:
it is for mapping only

<mapping>
    <target>discovery/config/create@lib${NAME}</target>
    <url>^/api/v1/admin/discovery/auto-discovery-config$</url>
    <method>PUT</method>
</mapping>